### PR TITLE
Removed docker plugin from Client

### PR DESCRIPTION
**Pull request by marcodiri**

### CODE REVIEW

File: java_socketio_chatroom_client/src/it/java/io/github/marcodiri/java_socketio_chatroom_client/view/swing/ClientSwingViewIT.java [hunk 1]
```diff
@@ -65,9 +65,9 @@ public void onSetUp() {
 
 	@Override
 	public void onTearDown() throws Exception {
+		client.disconnect();
```
Comment (lorenzobucci) [inline java_socketio_chatroom_client/src/it/java/io/github/marcodiri/java_socketio_chatroom_client/view/swing/ClientSwingViewIT.java]: I don't think we can call directly client's methods. These are view tests so I think we need to click on the disconnect button, like in e2e tests.

Comment (marcodiri) [reply java_socketio_chatroom_client/src/it/java/io/github/marcodiri/java_socketio_chatroom_client/view/swing/ClientSwingViewIT.java]: Should I remove the line altogether like it was before?

Comment (lorenzobucci) [reply java_socketio_chatroom_client/src/it/java/io/github/marcodiri/java_socketio_chatroom_client/view/swing/ClientSwingViewIT.java]: Or better add a click on disconnect button `window.button(JButtonMatcher.withText("Disconnect")).click();` at the end of each test. The same thing we currently do in connection. For the last test it doesn't matter.

Comment (marcodiri) [reply java_socketio_chatroom_client/src/it/java/io/github/marcodiri/java_socketio_chatroom_client/view/swing/ClientSwingViewIT.java]: Yeah but I didn't want to add another GUI click extending the testing time even more. Also after clicking disconnect I think the "disconnect" event doesn't even make it in time before the server is stopped.

Comment (lorenzobucci) [reply java_socketio_chatroom_client/src/it/java/io/github/marcodiri/java_socketio_chatroom_client/view/swing/ClientSwingViewIT.java]: Ok ok you're right. I didn't remember that the server is restarted at each test. So maybe you can leave it blank like before.

**Pull request merged**